src/extension/csw/core/src/test/java/org/geoserver/csw/CSWSimpleTestSupport.java

```diff
@@ -17,6 +17,7 @@
     protected void setUpTestData(SystemTestData testData) throws Exception {
         testData.setUpSecurity();
         System.setProperty("DefaultCatalogStore", "org.geoserver.csw.store.simple.GeoServerSimpleCatalogStore");
+        System.setProperty("ALLOW_ENV_PARAMETRIZATION", "true");
```

Comment:
Probably best to set it in the base test class, and make sure it gets cleaned at the end of the test?